### ISSUE TITLE
Modes and Keyboard backlight options cannot be interacted with

### ISSUE THREAD

**Issue by Zeperox** [dmitry-s93/MControlCenter#205]
Basically I can't even click on the buttons

**Comment by mutchiko** [comment]
@Zeperox what is your laptop model? does it have RGB lights?

**Comment by Zeperox** [comment]
@mutchiko MSI Leopard gl65 yeah it has rgb

Now that I think about it... I replaced my keyboard to one I found in AliExpress because the original laptop keyboard broke, that probably could be related in a way?

**Comment by mutchiko** [comment]
@Zeperox RGB keyboards can't be controlled by EC; they are connected by USB, so they can't be supported. try openRGB (it works for most msi laptops)

what version of MCC do you use? latest version will hide the page automatically if the keyboard lights aren't supported.

**Comment by Zeperox** [comment]
@mutchiko I think it's 0.4.1

**Comment by mutchiko** [comment]
@Zeperox i don't think that your laptop is supported, better to stay on the current version or open an issue to add support for your laptop in msi-ec
